bin/src/main.rs

```diff
@@ -9,6 +9,7 @@ use futures::Stream;
 use crate::stream_adapter::{StrictOrLazyLineBuilder, StrictOrLazyLines};
 use config::Config;
 use env_logger::Env;
+use fs::tail::Lookback;
```

Comment:
_Adding the comment here for convenience but the comment is not related to this line_

Should we keep `bin/default.profraw` out of source control?